src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -201,17 +201,19 @@ enum {
     UCT_IB_MLX5_MD_FLAG_UAR_USE_WC           = UCS_BIT(17),
     /* Device supports implicit ODP with PCI relaxed order */
     UCT_IB_MLX5_MD_FLAG_GVA_RO               = UCS_BIT(18),
-    /* RoCE supports out-of-order RDMA for RC */
+    /* Device supports out-of-order RDMA for RC */
     UCT_IB_MLX5_MD_FLAG_DP_ORDERING_OOO_RW_RC = UCS_BIT(19),
-    /* RoCE supports out-of-order RDMA for DC */
+    /* Device supports out-of-order RDMA for DC */
     UCT_IB_MLX5_MD_FLAG_DP_ORDERING_OOO_RW_DC = UCS_BIT(20),
-    /* RoCE supports forcing ordering configuration */
+    /* Device supports forcing ordering configuration */
     UCT_IB_MLX5_MD_FLAG_DP_ORDERING_FORCE     = UCS_BIT(21),
-    /* Device supports DDP (OOO data placement)*/
-    UCT_IB_MLX5_MD_FLAG_DDP                   = UCS_BIT(22),
+    /* Device supports direct data placement for RC */
+    UCT_IB_MLX5_MD_FLAG_DDP_RC                = UCS_BIT(22),
```

Comment:
maybe we should unify both namings (ddp / dp_ordering_ooo)

Reply:
The issue is they are entirely different capabilities we need somehow to distinguish between, they are queried differently